src/dali_backend.cc

```diff
@@ -21,7 +21,9 @@
 // SOFTWARE.
 
 #include "src/dali_backend.h"
+
```

Comment:
In triton coding convention we put a blank line here after the corresponding header file (if any)....

Reply:
This (and the one below) is actually what formatter gave me. I tried to rerun it multiple times, but it still insists on having those 2 blank lines...

Reply:
Did you run format.py with python or python3?